src/components/new-request/search.vue

```diff
@@ -564,17 +542,29 @@ export default class NewSearch extends Mixins(CommonMixin) {
     this.setNoCorpNum(false)
   }
 
-  @Watch('request_action_cd')
-  watchRequestActionCd (newVal: NrRequestActionCodes) {
-    // Set default location to BC for the requests where BC is the only location option
-    if ([NrRequestActionCodes.CONVERSION, NrRequestActionCodes.MOVE].includes(newVal)) {
+  /** Called when Request Action menu item is changed. */
+  onRequestActionChange (text: string): void {
+    this.request = RequestActions.find(request => request.text === text)
```

Comment:
Find and store the request action that matches the selected menu text. I couldn't figure out how to get the whole request object, or the group + value, from the v-select menu.